store/storetest/file_info_store.go

```diff
@@ -32,7 +32,7 @@ func TestFileInfoStore(t *testing.T, ss store.Store) {
 	t.Run("GetFilesBatchForIndexing", func(t *testing.T) { testFileInfoStoreGetFilesBatchForIndexing(t, ss) })
 	t.Run("CountAll", func(t *testing.T) { testFileInfoStoreCountAll(t, ss) })
 	t.Run("GetStorageUsage", func(t *testing.T) { testFileInfoGetStorageUsage(t, ss) })
-	t.Run("GetUptoNSizeFileTime", func(t *testing.T) { testGetUptoNSizeFileTime(t, ss) })
+	t.Run("GetUptoNSizeFileTime", func(t *testing.T) { testGetUptoNSizeFileTime(t, ss, s) })
```

Comment:
I had thought about registering a

```Go
t.Cleanup(func(){
   s.GetMasterX().Exec("TRUNCATE FileInfo")
})
```

at the beginning of this top-level test, so that each sub-test automatically gets cleaned up. I wasn't sure if this was too invasive, and decided to go with my approach in the other closed PR.

Reply:
Ah that's indeed a lovely idea!